client/src/components/ChartPopup/ChartPopup.css

```diff
@@ -0,0 +1,33 @@
+.chart-popup {
+  height: 100%;
+  width: 100%;
+  position: fixed;
+  top: 0;
```

Comment:
Please add the unit: px. There's quite a few places in this diff that are also missing the unit.

Reply:
Oh, I should have commented that I did not add units b/c all the other top, left, right, and bottom did not have any units.